infra/dlv/dlv-setup.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# Copyright 2016-2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.!/bin/bash
+
+SSH="ssh -o StrictHostKeyChecking=no"
+SCP="scp -q -o StrictHostKeyChecking=no"
```

Comment:
Is there a reason we have `-q` here, but not in the file above?

https://github.com/vmware/vic/blob/cf361bae4463b040377d8f81b8e7a156247bc6fa/infra/dlv/dlv-ctl.sh#L17